### ISSUE TITLE
"edit" icon missing

### ISSUE THREAD

**Issue by burner1024** [kulttuuri/DiscordNotifications#63]
V1.12, for some reason pretty "edit" icon is missing:
![Captura de pantalla de 2020-03-10 04-08-04](https://user-images.githubusercontent.com/1279852/76257783-f000a280-6284-11ea-9cbe-cf9a9dc2668e.png)
What should I check?

**Comment by kulttuuri** [comment]
Those are actually emojis, pencil icon is: 📝 :pencil:. I think that you have missing that emoji in your operating system (what OS are you using or was this on mobile?) or you have somehow disabled emojis in Discord or something like this.

**Comment by burner1024** [comment]
> I think that you are missing that emoji in your operating system (what OS are you using or was this on mobile?)

Linux.

> Does others see the emoji or are you the only one missing it?

Uhm, I don't know.

If I just copy/paste `:pencil:` into Discord message window, it displays properly:
![Captura de pantalla de 2020-05-08 15-01-39](https://user-images.githubusercontent.com/1279852/81385073-eee0da00-913c-11ea-855f-3223ef986f7e.png)

**Comment by kulttuuri** [comment]
Interesting. Is it linux desktop version, that could have bug with rendering emojis... Can you check with mobile or web version of Discord to verify this?

**Comment by burner1024** [comment]
It's exactly the same in web version (Firefox), icon missing.

**Comment by kulttuuri** [comment]
Hmm... 🤔 Not sure what is causing that then...

**Comment by kulttuuri** [comment]
Btw did that work before or it has always been missing?

**Comment by burner1024** [comment]
Emojis always worked, the one from wiki hoo - never.
Is there a way to log/see the payload that's being sent to Discord?

**Comment by kulttuuri** [comment]
Yeah, I was also wondering if the message sent from your MediaWiki server is being encoded in some weird format. To log the payload to PHP error log file, you can:
- Open file DiscordNotificationsCore.php
- Add this after line 511: `error_log($post);`
- View your PHP error log file (location depends on your installation of PHP)

**Comment by burner1024** [comment]
OK, so it's what's being sent: `[08-May-2020 08:52:33 UTC] {"embeds": [{ "color" : "2993970" ,"description" : "???? [Magus](https://`
PHP user locale is UTF8, db is in utf8mb4, everything is utf8 in pretty much any place I can think of.

What else can I check?

**Comment by kulttuuri** [comment]
Ah! So that is the issue. What method do you use the send the data, cURL?

**Comment by burner1024** [comment]
yes, curl

**Comment by kulttuuri** [comment]
If you open the language file `i8n/en.json` of the extension in your mediawiki installation, do you see messed up emojis or proper emojis? Maybe that file got messed up at some point or has wrong character encoding?

**Comment by burner1024** [comment]
It does look good
![Captura de pantalla de 2020-05-08 16-44-43](https://user-images.githubusercontent.com/1279852/81393894-4b4af600-914b-11ea-9440-7d0ec4501e53.png)

```
$ file extensions/DiscordNotifications/i18n/en.json 
extensions/DiscordNotifications/i18n/en.json: UTF-8 Unicode text
```

**Comment by kulttuuri** [comment]
I am not sure what the issue might be then... 🤔
